package.json

```diff
@@ -21,10 +21,9 @@
     "compile-zksync": "COMPILE_ZK=true yarn hardhat compile",
     "lint": "yarn lint-solidity && yarn lint-js && yarn lint-rust",
     "lint-solidity": "yarn solhint ./contracts/**/*.sol",
-    "lint-js": "yarn prettier --list-different && yarn lint-rust",
-    "lint-rust": "cargo fmt --all -- --check && cargo clippy",
-    "lint-fix": "yarn prettier --write && cargo fmt --all",
-    "prettier": "prettier .",
+    "lint-js": "yarn prettier --list-different **/*.js **/*.ts",
+    "lint-rust": "yarn prettier --list-different **/*.rs",
```

Comment:
this is quite slow locally as it tries to lint all anchor expanded macros. maybe we limit this to programs?

Reply:
sure, done.